### ISSUE TITLE
Alpha branch cnode build all script broken

### ISSUE THREAD

**Issue by huths0lo** [cardano-community/guild-operators#1441]
**Describe the bug**
I'm using the alpha branch to update my testnet nodes to 1.35.0.  At a certain point it is trying to copy a file that it cant find, and ending the script without finishing.

**To Reproduce**
1. Open $HOME/tmp
2. Run .prereq.sh -b alpha
3. Open ~/git/cardano-node
4. Run $CNODE_HOME/scripts/cabal-build-all.sh

**Expected behavior**
Should build Cardano Node and copy latest version of cardano-node and cardano-cli to system folder.

**Screenshots**
Symlinking 'bech32' to '/home/john/.cabal/bin/bech32'
Symlinking 'cardano-address' to '/home/john/.cabal/bin/cardano-address'
Symlinking 'cardano-ping' to '/home/john/.cabal/bin/cardano-ping'
Symlinking 'mux-demo' to '/home/john/.cabal/bin/mux-demo'
Copying setup to /home/john/.cabal/bin/

**-->cp: cannot stat '/home/john/git/cardano-node/dist-newstyle/tmp/src-85151/HsOpenSSL-0.11.7.2/dist/setup/setup': No such file or directory**

john@testnet-relay:~/git/cardano-node$ cardano-cli --version
cardano-cli 1.34.1 - linux-x86_64 - ghc-8.10
git rev 73f9a746362695dc2cb63ba757fbcabb81733d23

**Version:**
 - OS: Ubuntu 22.04lts
 - Product version: Alpha branch
 - Cardano Node version: Currently on 1.34.1
 - Network you're connecting to: testnet

**Comment by reqlez** [comment]
Did you ever figure this out after clearing the ${HOME}/.cabal/ +  ${HOME}/.ghcup/ directories?

I did hear that one SPO had to also upgrade their OpenSSL lib on the OS? But I had no issues with fresh 22.04 install.

Also, should run $CNODE_HOME/scripts/cabal-build-all.sh -o ( overwrite ) flag to overwrite the bins, no ?

**Comment by huths0lo** [comment]
Yes, clearing the cabal and ghc folders allowed the binaries to build.  Still working through issues with running the service.  I can run cnode.sh manually.  Its complaining about the env file.  But I've replaced it with the latest rev on the repo.

**Comment by rdlrt** [comment]
Why would you replace the file manually - unless you missed to add `-b alpha` to prereqs, your env would already be aligned.  Replacing manually could lead to loss of user defined variables.

The new issue you encounter could likely be due to the environment variables on shell vs systemd. "complaining about the env file" is a bit vague, could you add the error?

**Comment by huths0lo** [comment]
> Why would you replace the file manually - unless you missed to add `-b alpha` to prereqs, your env would already be aligned. Replacing manually could lead to loss of user defined variables.
> 
> The new issue you encounter could likely be due to the environment variables on shell vs systemd. "complaining about the env file" is a bit vague, could you add the error?

Theres only one modified variable, so I manually added it back in.  I took a backup of the old env before replacing it.

It looks like cnode is starting up fine now.  But cnode-dbsync is still complaining about the enviornment variables.  Below is the output from journalctl.

Jun 28 23:11:24 DS-Guild-Koios systemd[1]: Started Cardano DB Sync.
Jun 28 23:11:25 DS-Guild-Koios cnode-dbsync[465141]: Looks like cardano-node is running with socket-path as /opt/cardano/cnode/sockets/node0.socket, but the actual socket file does not exist.
Jun 28 23:11:25 DS-Guild-Koios cnode-dbsync[465141]: This could occur if the node hasnt completed startup or if a second instance of node startup was attempted!
Jun 28 23:11:25 DS-Guild-Koios cnode-dbsync[465141]: If this does not resolve automatically in a few minutes, you might want to restart your node and try again.
Jun 28 23:11:25 DS-Guild-Koios cnode-dbsync[465205]: TERM environment variable not set.

**Comment by huths0lo** [comment]
> Why would you replace the file manually - unless you missed to add `-b alpha` to prereqs, your env would already be aligned. Replacing manually could lead to loss of user defined variables.
> 
> The new issue you encounter could likely be due to the environment variables on shell vs systemd. "complaining about the env file" is a bit vague, could you add the error?

It seems like cnode is happy now with its systemd service.  Dbsync has the below error, which is very similar to what I was seeing with cnode when I first opened the issue.  Since I just restarted cnode under the service, I'll update once its back up.  The below may be moot at that point.

Jun 28 23:11:24 DS-Guild-Koios systemd[1]: Started Cardano DB Sync.
Jun 28 23:11:25 DS-Guild-Koios cnode-dbsync[465141]: Looks like cardano-node is running with socket-path as /opt/cardano/cnode/sockets/node0.socket, but the actual socket file does not exist.
Jun 28 23:11:25 DS-Guild-Koios cnode-dbsync[465141]: This could occur if the node hasnt completed startup or if a second instance of node startup was attempted!
Jun 28 23:11:25 DS-Guild-Koios cnode-dbsync[465141]: If this does not resolve automatically in a few minutes, you might want to restart your node and try again.
Jun 28 23:11:25 DS-Guild-Koios cnode-dbsync[465205]: TERM environment variable not set.
****
